dashboard/client/src/pages/overview/cards/RecentServeCard.tsx

```diff
@@ -20,33 +20,45 @@ type RecentServeCardProps = {
 export const RecentServeCard = ({ className }: RecentServeCardProps) => {
   const classes = useStyles();
 
-  // Use mock data by uncommenting the following line
-  // const applications = mockServeApplications.applications;
-  const { allServeApplications: applications } = useServeApplications();
+  const { allServeDeployments: deployments } = useServeDeployments();
```

Comment:
can we type some of the API response calls, such as `allServeDeployments`? it makes developing on the code hard for external ppl (like the `["application.last_deployed_time_s", "name"]` part on L27)

it may be worth looking into at some point if we can automatically export typescript models based on ray `@PublicAPI`

Reply:
It's typed right here: https://github.com/ray-project/ray/blob/master/dashboard/client/src/type/serve.ts#L112

I do want to do generated types from the API but we don't have the infra for that yet right now.